core/templates/pages/terms-page/terms-page.component.html

```diff
@@ -266,19 +266,19 @@ <h2>Changelog</h2>
     padding: 30px 55px 30px 45px;
   }
 
-  .oppia-page-cards-container .oppia-terms-right-menu {
+  .oppia-terms-right-menu {
```

Comment:
Can you find some way to keep all the CSS classes in this file scoped to this file only? E.g. prefix it with `terms-page`, or have a container div for the whole thing with an `oppia-terms-page` class, or something like that.

Reply:
I think this is not required for this component as it is already in angular 8. Styles of an angular 2+ component is scoped to the component only.

Reply:
Ah ok -- didn't realize! Approving, then -- thanks.